### PR TITLE
Added Contributing Documentation

### PR DESCRIPTION
**Pull request by KhuloodHassan**
### Fixes # (issue that will be closed by this PR)
- Issue CommunityFocus/CommunityFocus#7

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)

### Changes made
- Added contributing documentation

### Screenshots/ Screenshare
- N/A

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+3. Run `npm install` on both terminals - this will install all the dependencies to your local repository (you will see a node_modules folder pop up on each directory)
+4. Run `npm run dev` on both to run the app!
+
+### Making Changes
+
+You're now all set to work on your assigned issue!
+
+#### Changing Code
+
+Make changes like you would on any other project. Once you've made the chagnes and tested them thoroughly, you're ready to push changes to your fork!
```
Comment (nmpereira) [inline .github/CONTRIBUTING.md:84]:
```suggestion
Make changes like you would on any other project. Once you've made the changes and tested them thoroughly, you're ready to push changes to your fork!

```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+1. Make your changes
+2. Test thoroughly
+3. Run `npm run format <path-of-file(s)-you-changed>` - if you changed multiple files, please run this command separately for each file. This command runs prettier and formats the file(s) to Community Focus's configurations
+4. Run `npm run lint`
+5. Fix ESLint errors if any
+6. Commit your changes using `git commit -m "Commit-message"`
+7. Push your changes to your fork using `git push`
```
Comment (nmpereira) [inline .github/CONTRIBUTING.md:92]:
```suggestion
6. Run `npm run test` for any unit tests to run. They should all pass. If not, you need to see why the test failed and fix either the code, or update the tests to reflect your code changes.
7. Commit your changes using `git commit -m "Commit-message"`
8. Push your changes to your fork using `git push`
```

**Review by nmpereira — CHANGES_REQUESTED** [review]

**Comment by KhuloodHassan** [comment]
Revised with the changes!

**Review by nmpereira — APPROVED** [review]
LGTM! 🚀

**Comment by nmpereira** [comment]
@KhuloodHassan Approved it. Please go ahead and merge it. I would recommend using 'squash and merge'

**Pull request merged**